installing.md

```diff
@@ -1,7 +1,10 @@
 ## Recommended installation
 
+Wp-cli used phar archive. The phar archives are similar in concept to Java JAR archives. For more information [look this](http://php.net/manual/en/phar.using.intro.php).
+
 The recommended way to install WP-CLI is by downloading the Phar build, marking it executable, and placing it on your PATH.
```

Comment:
Instead of the additional sentence you've added, can we adapt this line?

> The recommended way to install WP-CLI is by downloading the Phar build (archives similar to Java JAR files, [see this article for more detail](http://php.net/manual/en/phar.using.intro.php)), marking it executable, and placing it on your PATH.